design-system/packages/core/src/components/Stack.tsx

```diff
@@ -67,7 +67,7 @@ export const Stack = forwardRefWithAs<'div', StackProps>(
           .filter(child => isValidElement(child))
           .map((child, index) => {
             return (
-              <Fragment>
+              <Fragment key={index}>
```

Comment:
not sure if `index` is right thing here but it does remove the error from console for now.

Reply:
should be fine here, thanks 👍